x-pack/plugins/canvas/public/components/font_picker/font_picker.tsx

```diff
@@ -32,7 +31,7 @@ export const FontPicker: FunctionComponent<Props> = props => {
     <EuiSuperSelect
       compressed
       options={displayedFonts.map(font => ({
-        value: font.value,
+        value: font.value as FontValue,
```

Comment:
@elastic/kibana-canvas this component states that arbitrary font values are supported, but the props and argument to the `onSelect` handler states that the value is a `FontValue`, not that it supports arbitrary strings... Can someone decide if it's better to ignore the fact that this supports arbitrary strings or should we switch away from `FontValue` for `string` types?